helm/chaos-mesh/templates/chaos-server-deployment.yaml

```diff
@@ -0,0 +1,93 @@
+{{- if .Values.chaosServer.create }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: chaos-server
+  labels:
+    app.kubernetes.io/name: {{ template "chaos-mesh.name" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/component: chaos-server
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+"  "_" }}
+spec:
+  replicas: {{ .Values.chaosServer.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ template "chaos-mesh.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/component: chaosServer
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ template "chaos-mesh.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/component: chaos-server
+    {{- with .Values.chaosServer.podAnnotations }}
+      annotations:
+{{ toYaml . | indent 8 }}
+    {{- end }}
+    spec:
+      {{- if .Values.chaosServer.serviceAccount }}
+      serviceAccount: {{ .Values.chaosServer.serviceAccount }}
+      {{- end }}
+      containers:
+        - name: chaos-server
+          image: {{ .Values.chaosServer.image }}
+          imagePullPolicy: {{ .Values.chaosServer.imagePullPolicy | default "IfNotPresent" }}
+          resources:
+{{ toYaml .Values.chaosServer.resources | indent 12 }}
+          command:
+            - /usr/local/bin/chaos-server
+          envFrom:
+          - configMapRef:
+            name: {{ template "chaos-mesh.name" . }}-chaos-server
+          volumeMounts:
+          - name: storage-volume
+            mountPath: {{ .Values.chaosServer.persistentVolume.mountPath }}
+            subPath: "{{ .Values.chaosServer.persistentVolume.subPath }}"
```

Comment:
Let's provide a set up with MySQL instead of sqlite?

Reply:
Using sqlite3 directly can speed up our current test and simple the steps of deployment, and we also allow to set MySQL DNS for chaos server store using the ENV in `value.yaml` file.

Reply:
Great